tracing-serde/src/lib.rs

```diff
@@ -25,6 +25,12 @@
     unused_parens,
     while_true
 )]
+// Support using Serde without the standard library!
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(not(feature = "std"))]
+use core::fmt;
+#[cfg(feature = "std")]
 use std::fmt;
```

Comment:
nit: `core` works in both, no need to duplicate the code. `std` is mostly a facade crate.
```suggestion
use core::fmt;
```

Reply:
we should probably do this in some of the other crates, which currently have their own weird facade modules to import names from either `std` or `core` conditionally. those are probably not necessary?